src/Microsoft.OData.Core/Json/IJsonWriter.cs

```diff
@@ -153,6 +156,15 @@ public interface IJsonWriter
         /// <param name="value">TimeOfDay value to be written.</param>
         void WriteValue(TimeOfDay value);
 
+#if NETCOREAPP3_1_OR_GREATER
+
+        /// <summary>
+        /// Write a <see cref="JsonElement"/> value.
+        /// </summary>
+        /// <param name="value">The <see cref="JsonElement"/> value to be written.</param>
+        void WriteValue(JsonElement value);

```

Comment:
If a property value is a JsonElement, what type is it?

Edm.JsonElement?

System.Text.Json.JsonElement?

Reply:
It's `System.Text.Json.JsonElement`. The `JsonElement` type isn't mapped to any specific EDM type.